frontend/src/components/auth/LoginForm.tsx

```diff
@@ -101,7 +106,18 @@ const LoginForm = ({
           onChange={(event) => setEmail(event.target.value)}
           isInvalid={invalidLogin}
           placeholder="Enter email address"
+          marginBottom="5px"
         />
+        {isSignup &&
+          (!checkValidEmail() ? (
+            <InfoAlert
+              message={SIGN_UP_INVALID_EMAIL_ALERT}
+              colour="orange.50"
+              height="55px"
+            />
+          ) : (
+            <Flex height="55px" />
```

Comment:
nice! ux nit: the empty space looks kinda weird - mind removing the conditional empty flex for the two alerts?
![image](https://user-images.githubusercontent.com/15113249/152083823-5e174a15-2c16-46ac-9655-32e9f8516717.png)

Reply:
how does this look?
![email validation](https://user-images.githubusercontent.com/32009013/152087327-3b746145-623b-4b53-b554-d106adb26256.gif)